ingestion-lambda/package.json

```diff
@@ -3,13 +3,19 @@
 	"version": "1.0.0",
 	"description": "",
 	"scripts": {
-		"dev": "tsx watch src/handler.ts",
+		"dev": "tsx watch localRun.ts",
 		"typecheck": "tsc -noEmit",
 		"build": "esbuild src/handler.ts --bundle --minify --outfile=dist/handler.js --external:aws-sdk --platform=node",
```

Comment:
might need to add `--external:@aws-sdk ` to this bundle command? (to ensure we exclude everything aws-sdk)

Reply:
in the build command there's already `--external:aws-sdk` -- do you mean we should replace this with `--external:@aws-sdk`, or have both?

Reply:
both, covers all bases